app/packages/looker/src/lookers/image.ts

```diff
@@ -21,11 +31,21 @@ export class ImageLooker extends AbstractLooker<ImageState> {
       ...options,
     };
 
+    // if in dynamic groups mode, add < > shortcuts, too
+    let shortcuts = { ...COMMON_SHORTCUTS };
+    if (config.isDynamicGroup) {
+      shortcuts = {
+        ...COMMON_SHORTCUTS,
+        previousFrameNoOpControl,
```

Comment:
these have explicit references to imavid - is that synonymous with dynamic groups?

Reply:
Yes 👍